qtawesome/tests/test_qtawesome.py

```diff
@@ -64,44 +64,78 @@ def test_bundled_font_user_installation():
         "powershell.exe",
         r'Get-ItemProperty "HKCU:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Fonts"',
     ]
-    fonts_result = subprocess.run(
-        fonts_command, capture_output=True, check=True, text=True
-    ).stdout
+    fonts_result = (
+        subprocess.run(fonts_command, capture_output=True, check=True, text=True)
+        .stdout.replace("\n", "")
+        .replace(" ", "")
+    )
     for font_filename in fonts_expected:
         assert font_filename in fonts_result
 
 
+def test_get_fonts_info():
+    """
+    Test that you can get the info of all the bundled fonts.
+    """
+    fonts_expected = [
+        font_filename
+        for _prefix, font_filename, _charmap_filename in qta._BUNDLED_FONTS
+    ]
+    fonts_root_dir, fonts_list = qta.get_fonts_info()
+    assert os.path.normcase(fonts_root_dir) == os.path.normcase(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "fonts"
+        )
+    )
+    assert set(fonts_list) == set(fonts_expected)
+
+
 @pytest.mark.skipif(os.name != "nt", reason="Only meant for Windows")
 def test_bundled_font_system_installation():
     """
     Test that the bundled fonts can be installed on Windows for all users.
 
-    See spyder-ide/qtawesome#167 and spyder-ide/spyder#18642
+    See spyder-ide/qtawesome#244
+
+    Notes:
+        * When running this test is possible that a prompt for privileges
+        appears(UAC prompt).
     """
     qta.install_bundled_fonts_system_wide()
     fonts_expected = [
-        "FontAwesome",
-        "codicon",
-        "elusiveicons",
-        "Font Awesome 5 Brands Regular",
-        "Font Awesome 5 Free Regular",
-        "Font Awesome 5 Free Solid",
-        "Material Design Icons 5.9.55 Regular",
-        "Material Design Icons",
-        "Phosphor",
-        "remixicon",
+        font_filename
+        for _prefix, font_filename, _charmap_filename in qta._BUNDLED_FONTS
     ]
     assert len(fonts_expected) == len(qta._BUNDLED_FONTS)
     fonts_command = [
         "powershell.exe",
         r'Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Fonts"',
     ]
-    fonts_result = subprocess.run(
-        fonts_command, capture_output=True, check=True, text=True
-    ).stdout
+    fonts_result = (
+        subprocess.run(fonts_command, capture_output=True, check=True, text=True)
+        .stdout.replace("\n", "")
+        .replace(" ", "")
+    )
     for font_filename in fonts_expected:
         assert font_filename in fonts_result
 
 
+def test_font_load_from_system_fonts(monkeypatch):
+    """
+    Test that the bundled fonts can be accessed from the system fonts on Windows.
```

Comment:
```suggestion
    Test that the bundled fonts can be accessed from the system fonts folder on
    Windows.
```

Clarify a bit more what the test does.